packages/lazy-fn-vite-plugin/tests/transpileSource.test.ts

```diff
@@ -32,46 +17,33 @@ type ViteTransformHandler = Exclude<
 function getTransform(): (...args: Parameters<ViteTransformHandler>) => string {
   return LazyLoadingPlugin().transform as never
 }
-
-describe('Transpile tuono source', () => {
-  it('Into the client bundle', () => {
-    const pluginTransform = getTransform()
-    const bundle = pluginTransform(SOURCE_CODE, 'id')
-    expect(bundle)
-      .toBe(`import { createRoute, lazyLoadComponent as dynamic } from 'tuono';
-const IndexImport = dynamic(() => import('./../src/routes/index'));
-const PokemonspokemonImport = dynamic(() => import('./../src/routes/pokemons/[pokemon]'));`)
-  })
-
-  it('Into the server bundle', () => {
-    const pluginTransform = getTransform()
-    const bundle = pluginTransform(SOURCE_CODE, 'id', {
-      ssr: true,
-    })
-    expect(bundle).toBe(`import { createRoute } from 'tuono';
-import IndexImport from "./../src/routes/index";
-import PokemonspokemonImport from "./../src/routes/pokemons/[pokemon]";`)
-  })
-})
-
-describe('Non tuono dynamic function', () => {
-  it('Into the client bundle', () => {
-    const pluginTransform = getTransform()
-    const bundle = pluginTransform(NON_DYNAMIC_SOURCE, 'id')
-    expect(bundle).toBe(`import { createRoute } from 'tuono';
-import { dynamic } from 'external-lib';
-const IndexImport = dynamic(() => import('./../src/routes/index'));
-const PokemonspokemonImport = dynamic(() => import('./../src/routes/pokemons/[pokemon]'));`)
-  })
-
-  it('Into the server bundle', () => {
-    const pluginTransform = getTransform()
-    const bundle = pluginTransform(NON_DYNAMIC_SOURCE, 'id', {
-      ssr: true,
-    })
-    expect(bundle).toBe(`import { createRoute } from 'tuono';
-import { dynamic } from 'external-lib';
-const IndexImport = dynamic(() => import('./../src/routes/index'));
-const PokemonspokemonImport = dynamic(() => import('./../src/routes/pokemons/[pokemon]'));`)
-  })
+describe('"dynamic" fn', async () => {
+  const folderNames = await fs.readdir(process.cwd() + '/tests/sources')
+
+  it.each(folderNames)(
+    'should correctly build the "%s" dynamic fn',
+    async (folderName) => {
+      const testDirPath = `${process.cwd()}/tests/sources/${folderName}`
+
+      const sourceBuf = await fs.readFile(path.join(testDirPath, 'source.tsx'))
+      const source = sourceBuf.toString()
```

Comment:
Specifying 'utf-8' as second `readFile` parameter directly returns a string, without the need to an additional variable.

```suggestion
      const source = await fs.readFile(
        path.join(testDirPath, 'source.tsx'),
        'utf-8',
      )
```